internal/dag/builder.go

```diff
@@ -518,6 +521,18 @@ func (b *Builder) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 			svhost.Secret = sec
 			svhost.MinProtoVersion = annotation.MinProtoVersion(proxy.Spec.VirtualHost.TLS.MinimumProtocolVersion)
 
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate && !isBlank(b.FallbackCertificate) {
+				secretName := splitSecret(b.FallbackCertificate, "projectcontour")
+				sec, err = b.lookupSecret(secretName, validSecret)
+				if err != nil {
+					sw.SetInvalid("FallbackCertificate requested, but Certificate Secret %q is invalid or missing", b.FallbackCertificate)
+					return
+				} else {
+					svhost.FallbackCertificate = sec
+					svhost.EnableFallbackCertificate = true
+				}
+			}
+
```

Comment:
I don't think the fallback certificate is compatible with client certificate auth, since when we have multiple routing tables behind a single filter chain, we can't enforce which routes get client auth and which don't. If we solve that problem, we could lift the restriction in the future.

Reply:
yup makes sense. I'll change it so you can't have both together in the same Root.